build/attestations/sbom.md

```diff
@@ -0,0 +1,233 @@
+    "documentNamespace": "https://anchore.com/syft/dir/run/src/core/sbom-cba61a72-fa95-4b60-b63f-03169eac25ca",
+    "name": "/run/src/core/sbom",
+    "packages": [
+      {
+        "SPDXID": "SPDXRef-b074348b8f56ea64",
+        "downloadLocation": "NOASSERTION",
+        "externalRefs": [
+          {
+            "referenceCategory": "SECURITY",
+            "referenceLocator": "cpe:2.3:a:org:repo:\\(devel\\):*:*:*:*:*:*:*",
+            "referenceType": "cpe23Type"
+          },
+          {
+            "referenceCategory": "PACKAGE_MANAGER",
+            "referenceLocator": "pkg:golang/github.com/org/repo@(devel)",
+            "referenceType": "purl"
+          }
+        ],
+        "filesAnalyzed": false,
+        "licenseConcluded": "NONE",
+        "licenseDeclared": "NONE",
+        "name": "github.com/org/repo",
+        "sourceInfo": "acquired package info from go module information: bin/server",
+        "versionInfo": "(devel)"
+      },
+      {
+        "SPDXID": "SPDXRef-1b96f57f8fed62d8",
+        "checksums": [
+          {
+            "algorithm": "SHA256",
+            "checksumValue": "0c13f1f3c1636491f716c2027c301f21f9dbed7c4a2185461ba94e3e58443408"
+          }
+        ],
+        "downloadLocation": "NOASSERTION",
+        "externalRefs": [
+          {
+            "referenceCategory": "SECURITY",
+            "referenceLocator": "cpe:2.3:a:go-chi:chi\\/v5:v5.0.0:*:*:*:*:*:*:*",
+            "referenceType": "cpe23Type"
+          },
+          {
+            "referenceCategory": "SECURITY",
+            "referenceLocator": "cpe:2.3:a:go_chi:chi\\/v5:v5.0.0:*:*:*:*:*:*:*",
+            "referenceType": "cpe23Type"
+          },
+          {
+            "referenceCategory": "SECURITY",
+            "referenceLocator": "cpe:2.3:a:go:chi\\/v5:v5.0.0:*:*:*:*:*:*:*",
+            "referenceType": "cpe23Type"
+          },
+          {
+            "referenceCategory": "PACKAGE_MANAGER",
+            "referenceLocator": "pkg:golang/github.com/go-chi/chi/v5@v5.0.0",
+            "referenceType": "purl"
+          }
+        ],
+        "filesAnalyzed": false,
+        "licenseConcluded": "NONE",
+        "licenseDeclared": "NONE",
+        "name": "github.com/go-chi/chi/v5",
+        "sourceInfo": "acquired package info from go module information: bin/server",
+        "versionInfo": "v5.0.0"
+      }
+    ],
+    "relationships": [
+      {
+        "relatedSpdxElement": "SPDXRef-1b96f57f8fed62d8",
+        "relationshipType": "CONTAINS",
+        "spdxElementId": "SPDXRef-043f7360d3c66bc31ba45388f16423aa58693289126421b71d884145f8837fe1"
+      },
+      {
+        "relatedSpdxElement": "SPDXRef-b074348b8f56ea64",
+        "relationshipType": "CONTAINS",
+        "spdxElementId": "SPDXRef-043f7360d3c66bc31ba45388f16423aa58693289126421b71d884145f8837fe1"
+      }
+    ],
+    "spdxVersion": "SPDX-2.2"
+  }
+}
+```
+
+## SBOM generator
+
+BuildKit generates the SBOM using a scanner plugin. By default, it uses is the
```

Comment:
I think we should probably include a disclaimer in this section, something to the effect of "you should always manually verify the results of your SBOM scan before publishing", maybe with some brief instructions of how to extract the info?

Reply:
I added a new section for verifying... Not sure I got it right though. PTAL!!

Reply:
Yup, I think that section is good :+1: Thanks!